drivers/SmartThings/zigbee-sensor/profiles/ias-generic-contact-sensor.yml

```diff
@@ -0,0 +1,14 @@
+name: ias-generic-contact-sensor
+components:
+- id: main
+  capabilities:
+  - id: contactSensor
+    version: 1
+  - id: batteryLevel
```

Comment:
battery level is different than battery.  All other Zigbee devices use the battery capability and that's what the defaults support not batteryLevel

Reply:
@varzac @tpmanley 
SRCN tried to use the battery capability but the result was untrustworthy.
So they removed the battery capabilty at the beginning, but I think If we cannot accurately display battery percentage, showing battery level (normal/warning/critical) instead would greatly benefit users.
SRCN suggested to use the zoneStatus(zone_status:is_battery_low_set()) because these devices are working based on IAS zone cluster. It is nice suggestion. Now, batteryLevel has 3 conditions.(normal/warning/critical) and we can know two battery condition with zoneStatus(1 – Low battery / 0 – Battery OK), so I think we can match the 'low battery' state to 'warning' state in batteryLevel. What do you think of this?

Reply:
I left a somewhat similar comment here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1921/files#r1956324647. 

Was the `BatteryPercentageRemaining` attribute found to not be trustworthy? We've found that usually works well, and the problems arise when trying to make `BatteryVoltage` to a percentage as that's very hard to do, especially for coin cell batteries that have a pretty flat voltage for most of their life and then quickly decrease at the end. I would suggest using `BatteryPercentageRemaining` and the `battery` capability and/or ZoneStatus low battery to indicate `warning` like you suggested (I don't have a strong preference whether it should be warning or critical).